src/logger.h

```diff
@@ -1,4 +1,28 @@
 #pragma once
+
+#include <fstream>
+
 #include <boost/log/trivial.hpp>
+#include <boost/log/utility/setup/from_stream.hpp>
+#include <boost/log/core/core.hpp>
+#include <boost/log/sources/logger.hpp>
+
+namespace logging = boost::log;
+namespace src = boost::log::sources;
+using LoggerType = src::logger_mt;
+
+#define CREATE_LOCAL_LOGGER(logger) \
```

Comment:
@AGaliuzov, what `logger` is used for? With hierarchical loggers (e.g. log4cxx) this is usually a logger name passed to the macro. But boost seems has no names.

Have you checked #4 ? There I've described how I see macroses, e.g. DECLARE_GET_LOGGER . So I want to return logger from a function.